TopMovies/View/CategoryVC/MoviesCategoryVC.swift

```diff
@@ -20,7 +20,10 @@ struct MoviesCategoryVCProps {
 
 // MARK: - VC class
 final class MoviesCategoryVC: UIViewController, PropsConnectable {
```

Comment:
Just mentioned that... Let's stick with a single name across the project.
It's either `***VC: UIViewController`, or `***ViewController: UIViewController`

Reply:
got it, thanks!